### ISSUE TITLE
use SUN/AccordionBox

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#100]
The pH meter in _Micro_ and _My Solutions_ screens should be using AccordionBox. For phet-io and a11y instrumentation.

**Comment by pixelzoom** [comment]
This would also address this odd structure for `phScale.microScreen.view.pHMeterNode`, where `expandCollapseButton` is currently a child of `valueNode`.

<img width="257" alt="screenshot_97" src="https://user-images.githubusercontent.com/3046552/73964545-9c8ae580-48cf-11ea-84de-f161b799182d.png">

**Comment by pixelzoom** [comment]
This is done. PHMeterNode has been converted to use AccordionBox.  

It is not (nor can it be) pixel-identical to the published version, but it looks fine to me.  @arouinfar please compare [1.4.0-dev.7]( https://phet-dev.colorado.edu/html/ph-scale/1.4.0-dev.7/phet/ph-scale_en_phet.html) to published version [1.3.4](https://phet.colorado.edu/sims/html/ph-scale/latest/ph-scale_en.html).  If this looks good visually, feel free to close. 

FYI, the new tandem structure is:

<img width="230" alt="screenshot_98" src="https://user-images.githubusercontent.com/3046552/73972275-c21eeb80-48dd-11ea-93a9-042da089ae06.png">

**Comment by arouinfar** [comment]
@pixelzoom visually the new PHMeterProbe is looking good and the tree structure seems reasonable.
